### DISCUSSION TITLE
Enable ligatures even if chars are different colors?

### DISCUSSION THREAD

**Discussion by georgeguimaraes** [wezterm/wezterm#4095] [General]
Hi,

Is there a way to enable ligatures when there's a difference in color between the characters?
It would be useful in Elixir now that Phoenix components use  `<.` a lot and colorschemes usually make them different colors.

![CleanShot 2023-08-04 at 17 54 01@2x](https://github.com/wez/wezterm/assets/2929/8e7e92d3-6f63-49a7-84ff-0e64a8a275c7)

**Comment by georgeguimaraes** [comment]
Actual code example:
![CleanShot 2023-08-04 at 18 30 12@2x](https://github.com/wez/wezterm/assets/2929/1b9af1ee-2691-457c-a48f-c90b507be7ce)

**Comment by wez** [comment]
wezterm doesn't have a way to support this. Ligatures are composed from runs of text that have the same attributes, and colors are attributes.

**Reply by georgeguimaraes** [reply]
I see, thanks for replying. 

I’ll try to change my colorscheme to achieve that.
